src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionBatchFixAllProvider.cs

```diff
@@ -108,15 +107,8 @@ internal abstract class AbstractSuppressionBatchFixAllProvider : FixAllProvider
                         args.progressTracker.ItemCompleted();
                     }
                 },
-                consumeItems: static async (stream, args, cancellationToken) =>
-                {
-                    await foreach (var tuple in stream)
-                        args.results.Add(tuple);
-                },
-                args: (@this: this, fixAllState, progressTracker, results),
+                args: (@this: this, fixAllState, progressTracker),
                 cancellationToken).ConfigureAwait(false);
-
-            return results.ToImmutableAndClear();
```

Comment:
the core helper can now just return an array.

Reply:
This change is great, I love it when a plan comes together!